### DISCUSSION TITLE
App users sidepanel presents only few users at Budibase version. 2.11.33.

### DISCUSSION THREAD

**Discussion by gmentzik** [Budibase/budibase#12133] [Help]
App users sidepanel presents only few users at Budibase version. 2.11.33.

**Comment by benaunt** [comment]
If you search for a user it will show them even if they are not present in the initial display. But the current implementation does need some work.

**Comment by melohagan** [comment]
Hey @gmentzik

If you have found a bug and it still persists as of **2.13**, please do raise a bug issue. Thanks!
